javaparser-testing/src/test/java/com/github/javaparser/ast/expr/NameTest.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2007-2010 Júlio Vilmar Gesser.
- * Copyright (C) 2011, 2013-2016 The JavaParser Team.
+ * Copyright (C) 2011, 2013-2017 The JavaParser Team.
```

Comment:
Nice find! We'll have to automate updating the headers when we have the generator stuff.